scripts/revert_changes_from_user.pl

```diff
@@ -181,38 +179,29 @@
 	# We have moved all revisions we don't want and have a list in %deleted_revs
 	# No update the product
 	if ($after_first_change) {
-		my $target = "$path/product.sto";
-		$target =~ s/\//_/g;
-		# keep a copy of current product
-		my $cmd = "mv $BASE_DIRS{PRODUCTS}/$path/product.sto $BASE_DIRS{REVERTED_PRODUCTS}/$target";
-		print STDERR "$code - $cmd\n";
-		# move does not work for symlinks on different file systems
-		#move("$BASE_DIRS{PRODUCTS}/$path/product.sto", "$BASE_DIRS{REVERTED_PRODUCTS}/$target") or die "Could not execute $cmd : $!\n";
+		# Just remove the original product link (no point copying it)
+		print STDERR "$code - removing product\n";
 		if (not $pretend) {
-			(system($cmd) == 0) or die "Could not execute $cmd : $!\n";
+			remove_object("$BASE_DIRS{PRODUCTS}/$path/product");
```

Comment:
This seems to be a change-in-behaviour - maybe worth relocating it into a separate PR?

Reply:
Sorry, that wasn't my intention. Looking above I think it should be a move

Reply:
Actually I've just had another look at the code. When we move a revision the file is renamed to something like 123_456_7890_1.sto (where the last "1" is the revision). However, the product.sto symlink will point to just "1.sto" so moving the existing product.sto file would just create a broken symlink so there doesn't seem any point in this. @stephanegigandet what do you think?